examples/vm_disk_example/variables.tf

```
@@ -24,3 +24,8 @@ variable "gcs_secret" {
   default     = ""
   description = "The secret name to pull from GCS bucket."
 }
+
+variable "gcs_images" {
+  type        = map(string)
+  description = "Map of the image name and GCS URL."
```

nit: Could we make it more clear the key vs value?
```suggestion
  description = "Map of the image name to GCS URL."
```